client/extensions/woocommerce/lib/order-values/index.js

```diff
@@ -53,6 +53,16 @@ export function getOrderFeeTotalTax( order ) {
 	return reduce( lines, ( sum, value ) => sum + getOrderFeeTax( order, value.id ), 0 );
 }
 
+/**
+ * Get the method title for the shipping value
+ *
+ * @param {Object} order An order as returned from API
+ * @return {String} Shipping method title
+ */
+export function getOrderShippingMethod( order ) {
+	return get( order, 'shipping_lines[0].method_title', false );
```

Comment:
We should get that information from the `labels` REST endpoint instead (think about porting this back to the plugin). If the user selected an USPS option at checkout, we already know which service he chose for each package. We could either add the price paid to that information or just get the "getOrderShippingMethod/getOrderShippingTotal" in PHP instead of in Calypso.